testing/hyperparams/test_get_set_hyperparams.py

```diff
@@ -23,7 +24,11 @@
 
 
 class SomeMetaStepMixin(NonTransformableMixin, NonFittableMixin, MetaStepMixin, BaseStep):
-    pass
+    def __init__(self, wrapped: BaseStep):
+        BaseStep.__init__(self)
+        NonTransformableMixin.__init__(self)
```

Comment:
The mixin should NOT call the base step constructor if his children already calls it. I think it was better to pass as before, a bug is probably introduced here.

Reply:
wrong. it crashes. 'has no attribute steps'

Reply:
no bug is introduced. just a bad name. should only be named SomeMetaStep